### ISSUE TITLE
Send `Connection: close` when we close a connection

### ISSUE THREAD

**Issue by 552020** [dantol29/webserver#327]
The default should be to keep the connection alive. We could eventually implement a timeout and close connection which are inactive.<br><br>NGNIX has a keep_alive timeout in the http block. 

```
http { 
    ... 
    keepalive_timeout 65; # Adjust the value as needed 
    ... 
}
```

@slombard said in [WEB-253](https://linear.app/web-serv/issue/WEB-253/redirect-traffic-from-the-browser-to-localhost#comment-72852a06):

> What does it mean 'we get some mozilla shit in the host header'. Mozilla can send all the shit it wants, but if we don't care about the headers send what's the problem. <br><br>This is another issue, but best practice from our side would be also to send a connection Close. In HTTP/1.1 apparently the keep alive is the standard, so we should not just close the connection when the response has been sent, but only after a sort of timeout where the client is not sending anything to us. But this is another issue.
